modules/videoio/cmake/detect_obsensor.cmake

```diff
@@ -13,8 +13,11 @@ if(NOT HAVE_OBSENSOR)
         set(HAVE_OBSENSOR TRUE)
         set(HAVE_OBSENSOR_ORBBEC_SDK TRUE)
         ocv_add_external_target(obsensor "${OrbbecSDK_INCLUDE_DIRS}" "${OrbbecSDK_LIBRARY}" "HAVE_OBSENSOR;HAVE_OBSENSOR_ORBBEC_SDK")
-        file(COPY ${OrbbecSDK_DLL_FILES} DESTINATION ${CMAKE_BINARY_DIR}/bin)
         file(COPY ${OrbbecSDK_DLL_FILES} DESTINATION ${CMAKE_BINARY_DIR}/lib)
+        if(NOT ORBBEC_SDK_VERSION STREQUAL "1")
+          # OrbbecSDK v2 loads some libraries at runtime.
+          file(COPY ${OrbbecSDK_RUNTIME_RESOURCE_FILES} DESTINATION ${CMAKE_BINARY_DIR}/lib)
+        endif()
         install(FILES ${OrbbecSDK_DLL_FILES} DESTINATION ${OPENCV_LIB_INSTALL_PATH})
```

Comment:
OrbbecSDK_RUNTIME_RESOURCE_FILES need to be installed to OPENCV_LIB_INSTALL_PATH for OrbbecSDK v2.